pandas/tests/groupby/test_function.py

```diff
@@ -1234,3 +1234,17 @@ def test_groupby_sum_timedelta_with_nat():
     res = gb["b"].sum(min_count=2)
     expected = Series([td3, pd.NaT], dtype="m8[ns]", name="b", index=expected.index)
     tm.assert_series_equal(res, expected)
+
+
+def test_groupby_empty_dataset():
+    # GH#41575
+    df = DataFrame(columns=["A", "B", "C"])
+    columns = ["count", "mean", "std", "min", "25%", "50%", "75%", "max"]
```

Comment:
Instead of hard coding here, you have a non-empty `df` do something like:

    result = df.iloc[:0].groupby("A").describe()
    expected = df.groupby("A").describe().iloc[:0]

Then this could test using kwargs percentiles / include / exclude. It would also have the benefit of ensuring consistent output even if the output of describe itself changes.

Reply:
Have updated